modules/core/arrow-core/src/main/kotlin/arrow/core/extensions/either.kt

```diff
@@ -94,7 +97,7 @@ interface EitherApply<L> : Apply<EitherPartialOf<L>>, EitherFunctor<L> {
 }
 
 @extension
-interface EitherApplicative<L> : Applicative<EitherPartialOf<L>>, EitherFunctor<L> {
+interface EitherApplicative<L> : Applicative<EitherPartialOf<L>>, EitherFunctor<L>, EitherApply<L> {
```

Comment:
If it has `EitherApply` than it doesn't need `EitherFunctor`. Neither here nor in the other extensions.

Reply:
`Apply` acts as an Extension to our current Typeclassopedia in `Arrow`
I discussed it in the `contributor` channel, whether to drop `Apply`, because it lacks in infrastructure in `Arrow`. After discussing and further research I am trying to give it a place in Arrow. 
![Screenshot 2019-08-26 at 12 21 24](https://user-images.githubusercontent.com/46971368/63684114-69123e00-c7fc-11e9-9784-ca585381d8ef.png)

Reply:
That's why we will keep both `<Datatype>Apply` and `<Data>`

Reply:
`interface Apply<F> : Functor<F>`

My point is that you already have a functor in the hierarchy of these classes via Apply, there's not need to have both.

Reply:
My Bad, we figured that the picture from above is wrong...

Reply:
Let me finish up with ApplyLaws and move forward